aries-site/src/pages/learn/the-box-model-part-one.mdx

```diff
@@ -0,0 +1,227 @@
+
+```
+<Box direction="row">
+  <Box width="large">
+    <Heading margin="none">
+      New HPE ProLiant Servers Engineered for your Hybrid World
+    </Heading>
+    <Text size="xlarge">
+      With a cloud operating experience, built-in security, and optimized
+      performance for your workloads to drive your business forward.
+    </Text>
+    <Button alignSelf="start" label="Learn more" kind="cta-primary" />
+  </Box>
+  <Box>
+    <Image src="https://www.insight.com/content/insight-web/en_US/shop/partner/hewlett-packard-enterprise/product-portfolio/servers/jcr%3Acontent/top-inner-full-width/column_layout/-column-1/image_0.img.jpg/1566699651505.jpg" />
+  </Box>
+</Box>
+```
+
+Note: Throughout this tutorial, we’ll be using t-shirt sizes. Make sure to read the [explanation on t-shirt sizes](/foundation/tshirt-sizing) for more information on what they are and their value in Grommet.
+
+At this point, your project should look something like this:
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/step-3.png" />
+</Box>
+
+## Add a background color and pad
+
+Now, let’s add a background color to the hero section by adding `background="background-contrast"` to the outer Box. We also want some space added between the edge of the outer Box and the content. To do so, add `pad="large"` to the outer Box.
+
+Remember that `pad` (equivalent to CSS padding) applies space _inside_ of a given container while `margin` applies space **\*\*\***outside**\*\*\*** of a given container.
+
+```
+<Box direction="row" background="background-contrast" pad="large">
+  <Box width="large">
+    <Heading margin="none">
+      New HPE ProLiant Servers Engineered for your Hybrid World
+    </Heading>
+    <Text size="xlarge">
+      With a cloud operating experience, built-in security, and optimized
+      performance for your workloads to drive your business forward.
+    </Text>
+    <Button alignSelf="start" label="Learn more" kind="cta-primary" />
+  </Box>
+  <Box>
+    <Image src="https://www.insight.com/content/insight-web/en_US/shop/partner/hewlett-packard-enterprise/product-portfolio/servers/jcr%3Acontent/top-inner-full-width/column_layout/-column-1/image_0.img.jpg/1566699651505.jpg" />
+  </Box>
+</Box>
+```
+
+At this point, your project should look something like this. We’re getting closer, but we’re missing the space between the various content elements.
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/step-3.png" />
+</Box>
+
+## Use gap to add space between elements
+
+Now, let’s add the spacing between elements. Whenever space is needed between elements, use `gap` on the parent Box of those elements. To add space between the image and the grouped content, add `gap="large"` to the outer Box.
+
+```
+<Box direction="row" background="background-contrast" pad="large">
```

Comment:
This code block is out of sync with the tutorial step. Missing the "gap" on the parent box.

Related to `<Example />` vs. images, hosting the code within a repository would keep these in sync. Instead of having to update this .mdx, we just point to the file on the repo.